### ISSUE TITLE
active see code for Signature 

### ISSUE THREAD

**Issue by carzamora** [pkp/textEditorExtras#9]
Hi, I think this plugin could active the option for see the code in the Signature field (Workflow Settings > Emails), for example to ad a line (hr), or use a <br> instead of <p>

**Comment by NateWr** [comment]
Good idea @carzamora! Would you be able to provide a pull request that implemented this change?

**Comment by carzamora** [comment]
sorry I am a biologist and journal editor with just a little knowledge in vanilla PHP

**Comment by NateWr** [comment]
No worries! I will put it on the list.
